e2e-tests/insurance/cypress/e2e/journeys/policy/single-contract-policy/total-contract-value/save-and-back.spec.js

```diff
@@ -78,33 +69,43 @@ context('Insurance - Policy - Single contract policy - Total contract value page
       cy.checkTaskPolicyStatusIsInProgress();
     });
 
-    describe('when going back to the page', () => {
-      beforeEach(() => {
-        cy.navigateToUrl(url);
+    it('should have the submitted values when going back to the page', () => {
+      cy.navigateToUrl(url);
+
+      fieldSelector(TOTAL_CONTRACT_VALUE).input().should('have.value', application.POLICY[TOTAL_CONTRACT_VALUE]);
```

Comment:
Is there not a command for this?

Reply:
Not yet.

Reply:
tech debt ticket to use checkValue or similar please

Reply:
Have created EMS-3850